### PR TITLE
Eliminating "unused symbol" warning

### PR DESCRIPTION
**Pull request by denis-bogdanas**

### CODE REVIEW

File: evm-types.md [hunk 1]
```diff
@@ -477,7 +477,7 @@ Most of EVM data is held in local memory.
     syntax Memory ::= Memory "[" Int ":=" ByteArray "]" [function, functional, klabel(mapWriteBytes)]
  // -------------------------------------------------------------------------------------------------
     rule WS [ START := WS' ] => replaceAtBytes(padRightBytes(WS, START +Int #sizeByteArray(WS'), 0), START, WS') requires START >=Int 0  [concrete]
-    rule WS [ START := WS':ByteArray ] => .Memory                                                                requires START  <Int 0  [concrete]
+    rule  _ [ START := _:ByteArray ] => .Memory                                                                requires START  <Int 0  [concrete]
```
Comment (ehildenb) [inline evm-types.md:480]:
```suggestion
    rule  _ [ START := _:ByteArray ] => .Memory                                                                  requires START  <Int 0  [concrete]
```

**Comment by daejunpark** [comment]
@denis-bogdanas Thanks for this. But still I got the unused symbol warning. Is it intended?

```
$ make build-haskell 
kompile --debug --backend haskell --md-selector "k & ( ( ! ( concrete | nobytes ) ) | symbolic | bytes   )" --hook-namespaces "KRYPTO JSON"  driver.md                     \
                   --directory .build/defn/./haskell -I /home/daejunpark/evm-semantics  \
                   --main-module ETHEREUM-SIMULATION     \
                   --syntax-module ETHEREUM-SIMULATION
[Warning] Compiler: Symbol '<block>' defined but not used. Add the 'unused'
attribute if this is intentional.
	Source(/home/daejunpark/evm-semantics/evm.md)
	Location(31,7,161,14)
[Warning] Compiler: Symbol '<ethereum>' defined but not used. Add the 'unused'
attribute if this is intentional.
	Source(/home/daejunpark/evm-semantics/evm.md)
	Location(31,7,161,14)
[Warning] Compiler: Symbol '<evm>' defined but not used. Add the 'unused'
attribute if this is intentional.
	Source(/home/daejunpark/evm-semantics/evm.md)
	Location(31,7,161,14)
[Warning] Compiler: Symbol '<kevm>' defined but not used. Add the 'unused'
attribute if this is intentional.
	Source(/home/daejunpark/evm-semantics/evm.md)
	Location(31,7,161,14)
[Warning] Compiler: Symbol '<network>' defined but not used. Add the 'unused'
attribute if this is intentional.
	Source(/home/daejunpark/evm-semantics/evm.md)
	Location(31,7,161,14)
[Warning] Compiler: Symbol 'ECDSASign(_,_)_KRYPTO_String_String_String' defined
but not used. Add the 'unused' attribute if this is intentional.
	Source(/home/daejunpark/evm-semantics/krypto.md)
	Location(28,23,28,104)
[Warning] Compiler: Symbol 'JSONnull' defined but not used. Add the 'unused'
attribute if this is intentional.
	Source(/home/daejunpark/evm-semantics/json.md)
	Location(18,24,18,72)
[Warning] Compiler: Variable 'WS' defined but not used. Prefix variable name
with underscore if this is intentional.
	Source(/home/daejunpark/evm-semantics/evm-types.md)
	Location(562,10,562,12)
```

**Comment by ehildenb** [comment]
@daejunpark these ones are being discussed/dealt with here: https://github.com/kframework/k/issues/1377

**Comment by denis-bogdanas** [comment]
The last one is actually a bug:
```
[Warning] Compiler: Variable 'WS' defined but not used. Prefix variable name
```

That variable is used.

**Comment by denis-bogdanas** [comment]
The last one should also be fixed, I mistakenly believed it belongs to a different rule. Opened another PR for this.

**Pull request merged**